build.rs

```diff
@@ -547,6 +551,12 @@ fn generate_bindings(r_paths: &InstallationPaths, version_info: &RVersionInfo) {
 
     // Finish the builder and generate the bindings.
     let bindings = bindgen_builder
+        .raw_line(format!(
+            "/* bindgen clang version: {} */",
+            bindgen::clang_version().full
+        ))
+        .raw_line(format!("/* clang-rs version: {} */", clang::get_version()))
+        .raw_line(format!("/* r version: {} */", version_info.full))
```

Comment:
Should we also write `libR-sys` version?

Reply:
Honestly, that's a very good idea.. I'd also like the `bindgen` version.

Reply:
Added the `libR-sys` version, but the `bindgen`-dependency version is not possible for me atleast. See https://github.com/rust-lang/cargo/issues/7846 if you're curious. I thought the `DEP_BINDGEN_PKG_VERSION` would work.